website/docs/secrets/setup-sops.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Setup Sops
+hide_title: true
+---
+
+
+# Managing secrets with Sops
+
+Mozilla SOPS is an open source tool for managing secrets. It uses a client-server approach to encrypting and decrypting the data key.
+
+In order to store secrets safely in a public or private Git repository, Flux's kustomize controller integrates with SOPS and the controller will decrypt SOPS-protected secrets during reconciliation.
+
+## Encrypting secrets using OpenPGP
+
+1- Generate a gpg key pairs
+
+```bash
+export KEY_NAME="gpg-key"
+export KEY_COMMENT="gpg key"
+
+gpg --batch --full-generate-key <<EOF
+%no-protection
+Key-Type: 1
+Key-Length: 4096
+Subkey-Type: 1
+Subkey-Length: 4096
+Expire-Date: 0
+Name-Comment: ${KEY_COMMENT}
+Name-Real: ${KEY_NAME}
+EOF
+```
+
+2- export the key pairs fingerprint in the shell
```

Comment:
it would be nice to show the output of step 1 and indicate which one is the fingerprint